ocamldoc/odoc_html.ml

```diff
@@ -2439,7 +2439,7 @@ class html =
 
         (* class inheritance *)
         self#generate_class_type_inheritance_info b clt;
-        (* a horizontal line *)
+        (* an horizontal line *)
```

Comment:
Me and google seem to agree than the "h" in "horizontal" is a consonant and therefore "a horizontal line" is the correct spelling.